cfn-resources/project-invitation/cmd/resource/resource.go

```diff
@@ -16,10 +16,10 @@ package resource
 
 import (
 	"context"
-	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Could you remove the space and run `go import`?
```suggestion
	"github.com/aws/aws-sdk-go/aws"
```